extensions/objects/templates/svg-filename-editor.component.spec.ts

```diff
@@ -435,15 +449,18 @@ describe('SvgFilenameEditor initialized with value attribute',
         value: 'svgimageFilename1.svg'
       });
       var mockDocument = document.createElement('div');
-      var strokeDiv = document.createElement('div');
-      strokeDiv.setAttribute('id', 'stroke-color');
-      var fillDiv = document.createElement('div');
-      fillDiv.setAttribute('id', 'fill-color');
-      var bgDiv = document.createElement('div');
-      bgDiv.setAttribute('id', 'bg-color');
-      mockDocument.appendChild(strokeDiv);
-      mockDocument.appendChild(fillDiv);
-      mockDocument.appendChild(bgDiv);
+      var colors = ['stroke', 'fill', 'bg'];
+      for (var i = 0; i < 3; i++) {
```

Comment:
This seems duplicated in multiple places. Can we make this a common helper function within the test file?

Reply:
Done.